api/utils/keypaths/keypaths.go

```diff
@@ -81,6 +90,9 @@ const (
 // ├── one.example.com.yaml            --> file containing profile details for proxy "one.example.com"
 // ├── two.example.com.yaml            --> file containing profile details for proxy "two.example.com"
 // ├── known_hosts                     --> trusted certificate authorities (their keys) in a format similar to known_hosts
+// ├── id_vnet                         --> SSH Private Key for third-party clients of VNet SSH
+// ├── id_vnet.pub                     --> SSH Public Key for third-party clients of VNet SSH
```

Comment:
Do we need to bother with the .pub file?

Reply:
Should we call it `vnet_authorized_keys` then? Or would that sort of imply that it would support all the extra bits in the `authorized_keys` format?

Reply:
I guess it would imply that and I don't really want to support all the extra authorized_keys format bits right now